keyboards/polykybd/tools/compare_codegen.py

```
@@ -0,0 +1,193 @@
+
+
+def data_symbols(elf: str, tool: str) -> "collections.Counter[tuple[str, int]]":
+    """Multiset of (name, size) for .data symbols.
+
+    A multiset, not a dict: file-scope statics in different translation units
+    share a name (this tree has FreeSansBold24pt7b three times and CSWTCH.5
+    twice), so keying by name alone collapses them and can hide a size change
+    in one behind its namesake.
+    """
+    nm = shutil.which(tool.replace("objdump", "nm")) or "nm"
+    res = subprocess.run([nm, "-S", elf], capture_output=True, text=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)